.gitignore

```diff
@@ -18,3 +18,5 @@ poetry.lock
 .mypy_cache/
 /dist/
 /test-results/
+/test.py
```

Comment:
Rather than adding this file to `.gitignore` for everyone, you can add to the file `.git/info/exclude` in the repository's folder on your machine. This will cause Git to ignore the file on your machine, without impacting others. 

`.gitignore` is only used for ignoring files that we expect others to also have on their machine.